### DISCUSSION TITLE
[Help]: Use align number

### DISCUSSION THREAD

**Discussion by sarussss** [davidjerleke/embla-carousel#765] [Help] [answered]
### Summary

Hi @davidjerleke,
How can I set the distance to 42px when dragging to the last slide?  The current gap is too big.
[Demo](https://stackblitz.com/edit/vitejs-vite-n7zags?file=index.html,style.css,main.js&terminal=dev)
https://tppr.me/JhvNeO

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [X] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### CodeSandbox example

https://stackblitz.com/edit/vitejs-vite-n7zags?file=index.html,style.css,main.js&terminal=dev

**Comment by davidjerleke** [comment]
Hi @sarussss,

Thanks for your question. I'm not sure what you want to achieve. Could you share an example from a different carousel library or try to elaborate?

Best,
David

**Reply by sarussss** [reply]
Hi @davidjerleke
Thank you for your reply.
I want the last slide to have a distance of only 42px like the first slide.
https://tppr.me/fvjwRi

**Reply by davidjerleke** [reply]
@sarussss so is it correct to assume that you want the first slide to have a distance of 42px to the left and the last slide 42px to the right?

**Reply by sarussss** [reply]
@davidjerleke Yes it is.

**Reply by davidjerleke** [reply]
@sarussss thanks for confirming. I will look into it when possible.

Best,
David

**Reply by sarussss** [reply]
@davidjerleke Thank you.

**Reply by davidjerleke** [reply, marked as answer]
@sarussss check this [StackBlitz](https://stackblitz.com/edit/vitejs-vite-bunzmy?file=index.html,style.css,main.js&terminal=dev) out and see if that's what you want to achieve. If yes, the simplest solution to this is not to do it with the `align` option, but using CSS instead:

```css
.embla__slide:first-child {
  margin-left: 42px;
}

.embla__slide:last-child {
  margin-right: 42px;
}
```

Best,
David

**Reply by sarussss** [reply]
@davidjerleke 

```css
.embla__slide:first-child {
  margin-inline-start: 42px;
}
.embla__slide:last-child {
  margin-inline-end: 42px;
}
```
Yes, I agree with this solution, but I think we can use this css so that when using rtl there is no need to rewrite the css. What do you think, will it affect Embla?
[Demo](https://stackblitz.com/edit/vitejs-vite-gogyfh?file=index.html,style.css,main.js&terminal=dev)

**Reply by davidjerleke** [reply]
@sarussss sure, if that's the requirement, to support both `LTR` and `RTL`, you can use the snippet you shared and it will work. Embla will still make correct calculations because Embla reads `computedStyles`. 

Example:

- Your markup is `LTR`
- `margin-inline-end: 42px;`
- Computed style translates into `margin-right: 42px;`

<img width="1427" alt="comupter" src="https://github.com/davidjerleke/embla-carousel/assets/11529148/e7cbe57a-4bc8-47ed-9511-380f5cdca813">

Best,
David

**Reply by sarussss** [reply]
@davidjerleke Thanks again!
